samples/matter/light_bulb/Kconfig

```
@@ -5,6 +5,22 @@
 #
 mainmenu "Matter nRF Connect Light Bulb Example Application"
 
+if MPSL_FEM
+
+config FEM_802_15_4_DEFAULT_TX_POWER
```

```suggestion
config DEFAULT_OPENTHREAD_TX_POWER
```
The name could suggest it's for setting TX gain on FEM itself.